tools/depends/native/libsquish-native/src/Makefile

```diff
@@ -1,6 +1,10 @@
 
 include config
 
+AR_FOR_BUILD ?= ar
+CXX_FOR_BUILD ?= cxx
```

Comment:
It works fine for my cross environment where we set these variables, and I only have a cross build environment available so I can't really test if anything I do breaks something when these variables don't get set.

As you can clearly see from the progress of these pull request I'm by no means an expert on this and could use any help/advice or guide into the right direction. I also don't mind if you just fix it yourself properly and close this pull request as I doubt I can be of much help.. sorry.